sdk/storage/Azure.Storage.DataMovement/api/Azure.Storage.DataMovement.net6.0.cs

```diff
@@ -178,8 +178,8 @@ public partial class StorageResourceReadStreamResult
     public partial class StorageResourceWriteToOffsetOptions
     {
         public StorageResourceWriteToOffsetOptions() { }
-        public string BlockId { get { throw null; } }
-        public long? Position { get { throw null; } }
+        public string BlockId { get { throw null; } set { } }
+        public long? Position { get { throw null; } set { } }
```

Comment:
this won't be a thing customers can ever mess with right?

Reply:
It won't be. But also this should have been public in the first place cause it's an options bag.